src/algebra/continued_fractions/theorems/continuants_recurrence.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2019 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import algebra.continued_fractions.theorems.translations
+/-!
+# Recurrence Lemmas for the `continuants` Function of Continued Fractions.
+
+## Summary
+
+Given a generalized continued fraction `g`, for all `n ≥ 1`, we prove that the `continuants`
+function indeed satisfies the following recurrences:
+
+  `Aₙ = bₙ * Aₙ₋₁ + aₙ * Aₙ₋₂`, and `Bₙ = bₙ * Bₙ₋₁ + aₙ * Bₙ₋₂`.
+-/
+
+namespace generalized_continued_fraction
+open generalized_continued_fraction as gcf
+variables {α : Type*} {g : gcf α} {n : ℕ} [division_ring α]
+
+lemma continuants_aux_recurrence {gp ppred pred : gcf.pair α}
+(nth_s_eq : g.s.nth n = some gp)
```

Comment:
This line and the next ones should all be indented: before the `:=`, everything is indented in mathlib.

Reply:
Fixed.